package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dropbox",
-  "version": "6.0.2",
+  "version": "6.1.0",
```

Comment:
I think this is a breaking change and would bust existing clients who are explicitly passing in `null` to routes that accept a Void param. Should just do a major change to 7.0.0.